build.sbt

```diff
@@ -341,6 +344,7 @@ lazy val commonDockerSettings = {
 
       List(dockerVersion, updateLatest, latestBranch, dockerTagName)
         .flatten
+        .flatMap(v => List(v, s"${v}_scala-${CrossVersion.binaryScalaVersion(scalaVersion.value)}"))
```

Comment:
I think we should be more explicit on which scala version is the default. Currently e.g. scala version in `latest`, `staging-latest` etc. is determined by build order?

Reply:
good point. now only build against `defaultScalaV` publishes images without `scalaV` suffix in tag.